pegasus/sites.v3/code.org/views/frozen_print_gallery.haml

```diff
@@ -39,6 +39,7 @@
     var printButton = $('#print_button');
     if (printButton) {
       printButton.click(function() {
+        trackEvent('frozen_print_gallery', 'print')
```

Comment:
One suggestion is to use 

```trackEvent('clicks', 'frozen_print_gallery', 'print')```

so that we avoid adding another top-level category when browsing with the GA UI.  

Also, nit: missing `;`.

Reply:
thank you for your expertise here, Brendan!